pytorch_lightning/accelerators/accelerator.py

```diff
@@ -439,6 +440,18 @@ def results(self) -> Any:
         """
         return self.training_type_plugin.results
 
+    @contextlib.contextmanager
+    def model_parallel_context(self) -> Generator:
```

Comment:
Shouldn't this be `model_sharded_context`

since we went with `configure_sharded_model`

not `configure_parallel_model`